### ISSUE TITLE
Mini search does not have <label>

### ISSUE THREAD

**Issue by lordscarlet** [umbraco/Umbraco-CMS#8706]
The primary search has a label, which is good for accessibility. However, the mini search does not.

## Umbraco version

I am seeing this issue on Umbraco version: 8.6.1

Reproduction
------------

### Bug summary

When searching the _Users_ (or any other) section, the mini search does not have a `<label>`

### Specifics

This effects anywhere with a mini-search, such as `/umbraco/#/users/users/users?filter=xyz`

![image](https://user-images.githubusercontent.com/17951/90908729-b939c700-e3a2-11ea-86d2-aea6e099382e.png)

### Expected result

The `<input>` should have an associated `<label>` with `for` attribute that matches the `id` of the `<input>`

### Actual result

There is no `<label>` or corresponding `id`.

**Comment by umbrabot** [comment]
Hi @lordscarlet,

We're writing to let you know that we would love some help with this issue. We feel that this issue is ideal to flag for a community member to work on it. Once flagged here, folk looking for issues to work on will know to look at yours. Of course, please feel free work on this yourself ;-).  If there are any changes to this status, we'll be sure to let you know. 

For more information about issues and states, have a look at [this blog post](https://umbraco.com/blog/the-umbraco-issue-tracker-process/)

Thanks muchly, from your friendly Umbraco GitHub bot :-)

**Comment by nikcio** [comment]
@nul800sebastiaan @lordscarlet I believe that this can be closed. Was just looking through some old issues and this one seems solved in #10583

**Comment by kjac** [comment]
Indeed this was fixed by https://github.com/umbraco/Umbraco-CMS/pull/10583 👍
